pkg/utils/sha256.go

```diff
@@ -0,0 +1,95 @@
+package utils
+
+func ComputeMessageSchedule(input []uint32) []uint32 {
+	// def compute_message_schedule(message: List[int]) -> List[int]:
+	// w = list(message)
+	// assert len(w) == 16
```

Comment:
we should assert this, or change the type to enforce the length

Reply:
Will add that check

Reply:
done!